cmd/gitops/config/options.go

```diff
@@ -1,11 +1,9 @@
 package config
 
 type Options struct {
-	Endpoint              string
-	OverrideInCluster     bool
-	GitHostTypes          map[string]string
-	InsecureSkipTLSVerify bool
-	Username              string
-	Password              string
-	Kubeconfig            string
+	Endpoint          string
+	OverrideInCluster bool
+	GitHostTypes      map[string]string
+	Username          string
+	Password          string
```

Comment:
I think these all _should_ really be in enterprise - except username and password is reused in the dashboard install command.